kinto/__init__.py

```diff
@@ -47,7 +47,8 @@ def main(global_config, config=None, **settings):
 
     # In Kinto API 1.x, a default bucket is available.
     # Force its inclusion if not specified in settings.
-    if 'kinto.plugins.default_bucket' not in settings['includes']:
+    if 'kinto.plugins.default_bucket' not in settings['includes'] \
```

Comment:
I would assign the key you're looking for into a variable first, and then use it in the comparisons.